internal/v1/api.yaml

```diff
@@ -680,6 +685,30 @@ components:
             $ref: '#/components/schemas/User'
           description:
             "list of users that a customer can add, also specifying their respective groups and SSH keys"
+    DistributionRepository:
+      type: object
+      required:
+        - id
+        - rhsm
+        - image_type_tags
+      properties:
+        id:
+          type: string
+          example: "baseos"
+        baseurl:
+          type: string
+          format: uri
+          example: "https://cdn.redhat.com/content/dist/rhel8/8/x86_64/baseos/os/"
+        metalink:
+          type: string
+          format: uri
+        rhsm:
+          type: boolean
+          example: true
+        image_type_tags:
+          type: array
+          items:
+            type: string
```

Comment:
I wonder if we can just reuse the existing Repository definition. The id isn't strictly required for anything, it's just for the package list script. And I don't think we need to return image_type_tags at this time.

Reply:
If we don't need id and tags, then reusing the existing Repository definition should be doable. I'll take a look at it

Reply:
Done!